jog_controller/launch/twist.launch

```diff
@@ -0,0 +1,19 @@
+<launch>
+  <arg name="joy_config" default="spacemouse_twist"/>
+  <arg name="group_name" default="manipulator"/>
+  <arg name="link_name" default="tool0"/>
+  <arg name="frame_id" default="tool0"/>
+  <arg name="sub_topic" default="spacenav/twist"/>
+
+  <include file="$(find spacenav_node)/launch/classic.launch"/>
```

Comment:
This makes CI fails. We should add dependency on spacenav_node in package.xml

Reply:
I couldn't figure out what was wrong, thanks! I'll fix it.